samples/Catalog/ios/Catalog/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAllowsArbitraryLoadsInWebContent</key>
```

Comment:
I actually don't think we even need this since we don't use a `WKWebView` in the Catalog. But doesn't hurt to keep it either.